STRATEGIC-INITIATIVES.md

```diff
@@ -0,0 +1,46 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  With the less-tangible approach to the work that the Community 
+Committee does, having a way to track this work is important.
+
+For each strategic initiative, the Community Committee's goal is to have a
+single point of reference - a champion - that understands what's happening
```

Comment:
Does this champion need to be a CommComm member/observer or not?
Wouldn't that be something worth being defined and written here??

Reply:
@Tiriel that's something I left open intentionally (see the section that needs champions) - we need to define that as a Committee before adding it in here, as I'm not sure if we've landed on a decision previously.

Reply:
Okay, at least I'm in tune with what has to be done :smile:

Reply:
Note that this CommComm document adds the additional specificity that a "champion" is a single point of reference to how the TSC describes it: https://github.com/nodejs/TSC/blob/master/Strategic-Initiatives.md

Reply:
I don't think we need to block on this, but coming up with a shared definition of `Champion` is worth doing. @MylesBorins has been the "champion" 😁 of defining the concept and the initiatives in the TSC.

Reply:
In that case, this may be a good agenda item for a TSC <> CommComm cross-committee meeting (ref: https://github.com/nodejs/admin/issues/26).